CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
   - Portuguese (pt-BR): am/pm
   - Romanian (ro): am/pm
   - Swedish (sv sv-FI sv-SE): am/pm
+  - Welsh (cy): Move the keys to rught place and add missing keys
```

Comment:
There is small typo here "rught" should "right"

Reply:
Sorry for it; it's already been fixed.